user-console/source/org/pentaho/mantle/client/MantleApplication.java

```diff
@@ -117,7 +117,11 @@ public native void setupNativeHooks( MantleApplication mantle, LoginCommand logi
               $wnd.addDataAccessGlassPaneListener(callback);
           }
           //CHECKSTYLE IGNORE LineLength FOR NEXT 1 LINES
-          mantle.@org.pentaho.mantle.client.MantleApplication::addGlassPaneListener(Lcom/google/gwt/core/client/JavaScriptObject;)(callback);
+          return mantle.@org.pentaho.mantle.client.MantleApplication::addGlassPaneListener(Lcom/google/gwt/core/client/JavaScriptObject;)(callback);
+      }
+
+      $wnd.removeGlassPaneListener = function (uuid) {
```

Comment:
Given that you are not actually accepting a GlassPaneListener object, and to keep it in line with the GWT-commons change, would it be better to name this function (and the private counterpart a few lines below) "removeGlassPaneListenerById"? 

Otherwise the next guy seeing that function will try to pass a listener-instance in and will be surprised by the errors he gets. A lack of a proper type system makes these pitfalls even more deadly.